tests/market/test_market_info.py

```diff
@@ -0,0 +1,282 @@
+    if opened_element.all() and opened_element.get_by_role("button").is_visible():
+        opened_element.get_by_role("button").click()
+
+
+def validate_info_section(page: Page, fields: [[str, str]]):
+    for rowNumber, field in enumerate(fields):
+        name, value = field
+        expect(
+            page.get_by_test_id("key-value-table-row").nth(rowNumber).locator("dt")
+        ).to_contain_text(name)
+        expect(
+            page.get_by_test_id("key-value-table-row").nth(rowNumber).locator("dd")
+        ).to_contain_text(value)
+
+
+def test_market_info_current_fees(page: Page):
+    # 6002-MDET-101
+    page.get_by_test_id(market_title_test_id).get_by_text("Current fees").click()
+    fields = [
+        ["Maker Fee", "10.00%"],
+        ["Infrastructure Fee", "0.05%"],
+        ["Liquidity Fee", "0.00%"],
+        ["Total Fees", "10.05%"],
+    ]
+    validate_info_section(page, fields)
+
+
+def test_market_info_market_price(page: Page):
+    # 6002-MDET-102
+    page.get_by_test_id(market_title_test_id).get_by_text("Market price").click()
+    fields = [
+        ["Mark Price", "107.50"],
+        ["Best Bid Price", "101.50"],
+        ["Best Offer Price", "103.50"],
+        ["Quote Unit", "BTC"],
+    ]
+    validate_info_section(page, fields)
+
+
+def test_market_info_market_volume(page: Page):
+    # 6002-MDET-103
+    page.get_by_test_id(market_title_test_id).get_by_text("Market volume").click()
+    fields = [
+        ["24 Hour Volume", "-"],
+        ["Open Interest", "1"],
+        ["Best Bid Volume", "99"],
+        ["Best Offer Volume", "96"],
+        ["Best Static Bid Volume", "1"],
+        ["Best Static Offer Volume", "1"],
+    ]
+    validate_info_section(page, fields)
+
+
+def test_market_info_insurance_pool(page: Page):
+    # 6002-MDET-104
+    page.get_by_test_id(market_title_test_id).get_by_text("Insurance pool").click()
+    fields = [["Balance", "0.00 tDAI"]]
+    validate_info_section(page, fields)
+
+
+def test_market_info_key_details(page: Page, vega: VegaService):
+    # 6002-MDET-201
+    page.get_by_test_id(market_title_test_id).get_by_text("Key details").click()
+    fields = [
+        ["Name", "BTC:DAI_2023"],
+        ["Market ID", vega.find_market_id("BTC:DAI_2023")],
+        ["Parent Market ID", "-"],
+        ["Insurance Pool Fraction", "-"],
+        ["Trading Mode", "Continuous"],
+        ["Market Decimal Places", "5"],
+        ["Position Decimal Places", "0"],
+        ["Settlement Asset Decimal Places", "5"],
+    ]
+    validate_info_section(page, fields)
+
+
+def test_market_info_instrument(page: Page):
+    # 6002-MDET-202
+    page.get_by_test_id(market_title_test_id).get_by_text("Instrument").click()
+    fields = [
+        ["Market Name", "BTC:DAI_2023"],
+        ["Code", "BTC:DAI_2023"],
+        ["Product Type", "Future"],
+        ["Quote Name", "BTC"],
+    ]
+    validate_info_section(page, fields)
+
+
+# @pytest.mark.skip("oracle test to be fixed")
```

Comment:
Need removing?

Reply:
will be fixed in next PR with oracle fixed in console